pkg/client/message.go

```diff
@@ -21,14 +21,14 @@ package client
 type Message struct {
 	// The message body, which is an a string.
 	// The ContentType indicates the format of this body.
-	Body string // Content of message
+	Data map[string]interface{} // Content of message
```

Comment:
We can:
a. use interface{} instead of map[string]interface{}
b. have another more low level funtion that take []byte

I can do both :-) witch one do you thinkg we should do (or another solution) ?

Reply:
I think that using `interface{}` is better for the short term.

For the long term `Message` should be an interface representing an immutable message, and there should be a mechanism to build messages that supports JSON or arrays of bytes.

Reply:
:+1: Thanks, On it